languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/added.txt

```diff
@@ -9193,4 +9193,11 @@ Verne	Verne	NPMS000
 chifruda	chifrudo	NCFS000
 chifrudas	chifrudo	NCFP000
 tórax	tórax	NCMN000
-
+homeostase	homeostase	NCFS000
+homeostases	homeostase	NCFP000
+crónico	crónico	AQ0MS0
+crónicos	crónico	AQ0MP0
```

Comment:
Can we please also tag `crónicas`, `crónica`, `crônica`, `crônico`, `crônicas`, and `crônicos` as adjectives? Just realized they are not tagged.

Reply:
@susanaboatto 

I can't find those words in the spelling.txt in the branch: [lt_marcoagpinto_20231105_0005](https://github.com/languagetool-org/languagetool/tree/lt_marcoagpinto_20231105_0005)

Maybe they were added in a previous branch?